modules/misunderstood/src/views/full/MainScreen/NewEventView.tsx

```diff
@@ -75,26 +76,31 @@ class NewEventView extends React.Component<Props, State> {
           New Misunderstood | {eventIndex + 1} of {totalEventsCount}
         </h3>
 
-        <ChatPreview messages={event.context} />
-
-        <h4 className={style.newEventPreview}>{event.preview}</h4>
-
-        <ButtonGroup large>
-          <Button
-            onClick={skipEvent}
-            icon="arrow-right"
-            intent={Intent.WARNING}
-            disabled={isAmending || eventIndex === totalEventsCount - 1}
-          >
-            Skip
-          </Button>
-          <Button onClick={deleteEvent} icon="trash" intent={Intent.DANGER} disabled={isAmending}>
-            Ignore
-          </Button>
-          <Button onClick={this.startAmend} icon="confirm" intent={Intent.PRIMARY} disabled={isAmending}>
-            Amend
-          </Button>
-        </ButtonGroup>
+        {!isAmending && (
+          <>
```

Comment:
What's that ? Does this works like `Ract.Fragment` ?

Reply:
yeah it seems like it does, was already in the file somewhere else so I reused it, not sure if it's a good practice or not, but it works 🤷‍♂